beacon_node/network/src/sync/block_lookups/mod.rs

```diff
@@ -990,20 +915,12 @@ impl<T: BeaconChainTypes> BlockLookups<T> {
                         lookup.request_block_and_blobs(cx)?
```

Comment:
is there a reason the line above is repeated twice?
```rust
lookup
    .block_request_state
    .state
    .register_failure_downloading();
lookup
    .blob_request_state
    .state
    .register_failure_downloading();
```

Reply:
one's updating the blob request state and one's updating the block request state

Reply:
oh derp i missed the 2 character difference 🤦